sweepbatcher/sweep_batch.go

```diff
@@ -462,6 +467,13 @@ func (b *batch) addSweep(ctx context.Context, sweep *sweep) (bool, error) {
 		return true, nil
 	}
 
+	// Enforce MaxSweepsPerBatch. If there are already too many sweeps in
+	// the batch, do not add another sweep to prevent the tx from becoming
+	// non-standard.
+	if len(b.sweeps) >= MaxSweepsPerBatch {
+		return false, nil
```

Comment:
does it make sense to log such an occurence?

Reply:
I added Info log message in cases when addSweep returns accept=false and err=nil in a separate commit "sweepbatcher: log the reason of acceptance failure".